Detectors/Upgrade/ITS3/base/include/ITS3Base/ContainerFactory.h

```diff
@@ -0,0 +1,53 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file ContainerFactory.h
+/// \brief Definition of the ContainerFactory class
+
+#ifndef ALICEO2_ITS_CONTAINERFACTORY_H_
+#define ALICEO2_ITS_CONTAINERFACTORY_H_
+
+#include "FairContFact.h" // for FairContFact, FairContainer (ptr only)
+#include "Rtypes.h"       // for ContainerFactory::Class, ClassDef, etc
+
+class FairParSet;
+
+class FairContainer;
+
+namespace o2
+{
+namespace its3
+{
+class ContainerFactory : public FairContFact
```

Comment:
Do we still need this? We are not really using this container/parameter feature of FairRoot.
(So essentially I am wondering if the code compiles and runs without this).

Reply:
You mean the whole file, right?
Now that I notice the `ALICEO2_ITS_CONTAINERFACTORY_H_` symbol is the same created in ITS, I forgot to convert it. I guess I can try without it.